webpack.config.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disabled */
+// eslint-disable-next-line import/no-extraneous-dependencies
 // Generated using webpack-cli https://github.com/webpack/webpack-cli
 
 const path = require('path');
```

Comment:
- [ ] To build up the React application, you should use "Create React App (CRA)" rather than "webpack," as per the project requirements. Learn more about creating a react app by visiting [this link](https://create-react-app.dev/docs/getting-started).
- Project requirements 👇:
<img width="646" alt="image" src="https://user-images.githubusercontent.com/77190203/166107371-cc64b194-2b11-4b05-ad80-6412e196113a.png">